src/gui/Src/BasicView/Disassembly.cpp

```diff
@@ -116,6 +116,8 @@ void Disassembly::updateColors()
 void Disassembly::updateFonts()
 {
     setFont(ConfigFont("Disassembly"));
+    delete mFontMetrics;

```

Comment:
Probably I can move that into `AbstractTableView` as a separate function, but not in `updateFonts` because there is another `setFont` to set the font for generic table views there.